Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoDecoderH264.mm

```diff
@@ -161,7 +161,8 @@ - (NSInteger)decodeData:(const uint8_t *)data
   rtc::ScopedCFTypeRef<CMVideoFormatDescriptionRef> inputFormat =
       rtc::ScopedCF(webrtc::CreateVideoFormatDescription(data, size));
   if (inputFormat) {
-    _reorderQueue.setReorderSize(webrtc::ComputeH264ReorderSizeFromAnnexB(data, size));
+    // FIXME: Reenable window computation size for AnnexB decoders, once feasible.
```

Comment:
Can you put the relevant Radar or Bugzilla ID in the FIXME?
(i.e., `FIXME(rdar://123456): Reenable window ...`)

Reply:
Done